llarp/util/endian.hpp

```diff
@@ -189,4 +190,19 @@ htole64buf(void *buf, uint64_t big64)
   htobuf64(buf, htole64(big64));
 }
 
+inline absl::uint128
+ntoh128(absl::uint128 i)
+{
+#if __BYTE_ORDER == __BIG_ENDIAN
+  return i;
+#else
+  uint64_t *ptr = (uint64_t *)&i;
```

Comment:
This can be rewritten more safely as something like:

```c++
auto lowSwapped = htobe64(Uint128Low64(i));
auto highSwapped = htobe64(Uint128High64(i));

return MakeUint128(lowSwapped, highSwapped);
```

Reply:
yeah

Reply:
oh hay those are the functions i wanted, fixing.